grails-app/controllers/org/broadinstitute/orsp/AuthenticatedController.groovy

```diff
@@ -62,7 +62,21 @@ class AuthenticatedController implements Interceptor, UserInfo {
 
     protected Collection<User> getPIsForIssue(Issue issue) {
         Collection<String> pis = IssueExtraProperty.findAllByProjectKeyAndName(issue.projectKey, IssueExtraProperty.PI)*.value
-        userService.findUsers(pis)
+        Collection<User> pisForUsers = userService.findUsers(pis) ?: new ArrayList<>()
+        if (pis.isEmpty()) {
+            User reporter = userService.findUser(issue.reporter)
+            if (reporter) { pisForUsers.add(reporter) }
+        }
+        pisForUsers
+    }
+
+    protected User getRequestorForIssue(Issue issue) {
+        userService.findUser(issue.reporter) ?: null
+    }
+
+    protected Collection<User> getCollaborators(Collection<String> collaboratorsList) {
+        Collection<User> collaborators = userService.findUsers(collaboratorsList) ?: new ArrayList<>()
+        collaborators
```

Comment:
I think this can be simplified to just `userService.findUsers(collaboratorsList)`